source/graphql/migrate-hasura.txt

```diff
@@ -37,63 +38,23 @@ Migrate to Hasura
 -----------------
 
 Migrating your GraphQL API endpoints from MongoDB Atlas App Services to Hasura
-involves a multi-step process that encompasses setting up your environment in
+is a multi-step process that encompasses setting up your environment in
 Hasura, configuring database connections, migrating schemas, and implementing
 authorization and authentication mechanisms. Below is an expanded guide
 detailing each step, with a focus on authorization and role-based access control
 (RBAC) within Hasura. For more information, check out the `Hasura docs 
-<https://hasura.io/docs/latest/databases/mongodb/mongodb-atlas/>`__.
+<https://hasura.io/docs/3.0/connectors/mongodb/>`__.
 
 To migrate to Hasura:
 
-#. :ref:`Create a new project in Hasura <hasura-create>`
 #. :ref:`Authorize Hasura for MongoDB <hasura-authorize-mongodb>`
-#. :ref:`Migrate your GraphQL schema <hasura-migrate-schema>`
+#. :ref:`Create a new project and connect to Hasura <hasura-create>`
 #. :ref:`Test your GraphQL queries <hasura-test>`
 #. :ref:`Handle authorization and authentication <hasura-auth>`
 #. :ref:`Set up custom resolvers <hasura-resolvers>`
 #. :ref:`Update client applications <hasura-client>`
 #. :ref:`Shut down MongoDB Atlas App Services <hasura-shutdown>`
 
-.. _hasura-create:
-
-Create a New Project in Hasura
-------------------------------
-
-.. procedure::
-
-   .. step::
-
-      After logging in to Hasura, navigate to Projects and create a new project
-      by clicking on :guilabel:`New Project`. Select a pricing plan, cloud
-      provider, and region that aligns with your MongoDB Atlas setup for optimal
-      performance.
-
-      .. image:: /images/migration/new-project.png
```

Comment:
It looks like we're removing a bunch of images from the page, but I don't see them removed in the PR. It would be good to remove them from the repo so unneeded, unmaintained files aren't hanging out indefinitely.

Reply:
Ah, yep. Thanks for the reminder!